Dockerfile-slim.template

```diff
@@ -1,4 +1,4 @@
-FROM python:%%PYTHON_VERSION%%-slim-stretch
+FROM python:%%PYTHON_VERSION%%-slim-buster
```

Comment:
I would also change the distribution of the PostgreSQL Apt repository.

Reply:
Good catch. Resolved in https://github.com/azavea/docker-django/pull/87/commits/f1f71707ecccc645ec4ca179fb56b72c0ff8b84a and https://github.com/azavea/docker-django/pull/87/commits/40707c54d1111cf408954438686b169e69997774.